src/components/Input.tsx

```diff
@@ -234,11 +238,21 @@ const InputStyle = css`
   }
 `;
 
-const $NumericFormat = styled(NumericFormat)`
+const $NumericFormat = styled(NumericFormat)<{ backgroundColorOverride?: string }>`
   ${InputStyle}
   font-feature-settings: var(--fontFeature-monoNumbers);
+  ${({ backgroundColorOverride }) =>
+    backgroundColorOverride &&
+    css`
+      background-color: ${backgroundColorOverride};
+    `}
 `;
 
-const $Input = styled.input`
+const $Input = styled.input<{ backgroundColorOverride?: string }>`
```

Comment:
use an ephemeral prop here 
```suggestion
const $Input = styled.input<{ $backgroundColorOverride?: string }>`
```

Reply:
or just use tailwind to see the background-color override?